usaspending_api/download/filestreaming/zip_file.py

```diff
@@ -3,9 +3,26 @@
 
 
 def append_files_to_zip_file(file_paths, zip_file_path):
+    """
+    Create zip archive at the specified zip_file_path if it does not exist, and add all the files at provided
+    file_paths to it.
 
-    # Append all of the provided files to the specified zip file.
+    NOTE: If a zip file already exists at zip_file_path, the given files will be added in addition to the ones
```

Comment:
No edits necessary. Writing thoughts


The function name `append_files_to_zip_file()`  leads me to believe appending to an existing file is the intended workflow. Recent changes to include timestamps in the data filenames should prevent duplicate file names even if a failure occurred during a previous attempt (unlike zip files which are stored in the DB and should remove any pre-existing file on collision).